apps/condo/domains/resident/schema/RegisterServiceConsumerService.js

```diff
@@ -104,8 +104,13 @@ const RegisterServiceConsumerService = new GQLCustomSchema('RegisterServiceConsu
                 const { data: { dv, sender, resident: { id }, accountNumber, paymentCategory } } = args
 
                 if (!accountNumber || accountNumber.length === 0) throw new GQLError(ERRORS.ACCOUNT_NUMBER_IS_NOT_SPECIFIED, context)
-
-                const resident = await Resident.getOne(context, { id, deletedAt: null }, 'id user { id } address addressKey')
+                const isAdmin = get(context, 'authedItem.isAdmin')
+                const residentWhere = {
```

Comment:
You need to check it in access

Reply:
Or it’s better always use the same logic. It should not depend on isAdmin access